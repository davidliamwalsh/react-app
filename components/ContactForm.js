import { Component } from 'react'

class ContactForm extends Component {
  render () {
    return <div className="c-contact__form--container">
      <div className="f-contact-form">
        <label for="message_name">Name</label>
        <input className="f-contact-form__input" placeholder="Your Name" type="text" name="message[name]" id="message_name" />
        <label for="message_email">Email</label>
        <input className="f-contact-form__input" placeholder="Your e-mail address" type="text" name="message[email]" id="message_email" />
        <label for="message_body">Body</label>
        <textarea rows="10" className="f-contact-form__input" placeholder="Please type your message" name="message[body]" id="message_body"></textarea>
      </div>
      <input type="submit" name="commit" value="Send" className="o-submit__btn" data-disable-with="Send" />
    </div>
  }
}

export default ContactForm