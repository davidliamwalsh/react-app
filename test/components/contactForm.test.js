import ContactForm from '../../components/ContactForm'

describe('<ContactForm />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <ContactForm />
    )

    expect(component).toMatchSnapshot()
  })
}) 