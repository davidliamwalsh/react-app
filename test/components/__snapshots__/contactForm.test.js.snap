// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactForm /> it renders correctly 1`] = `ShallowWrapper {}`;